pkg/controller/external_secrets/controller.go

```diff
@@ -457,5 +464,6 @@ func (r *Reconciler) cleanUp(esc *operatorv1alpha1.ExternalSecretsConfig, req ct
 	}
 	r.log.V(1).Info("removed finalizer, cleanup complete", "request", req.NamespacedName)
 
+	// TODO: Cleanup trusted CA bundle ConfigMap if it exists
```

Comment:
nit: Not mandatory to act. The TODO above covers all resources created for the operand installation, which would also include trusted CA ConfigMap.

Reply:
Removed this comment line.